posthog/api/team.py

```diff
@@ -64,7 +69,28 @@ class Meta:
             "created_at",
             "updated_at",
             "ingested_event",
+            "effective_membership_level",
+        )
+
+    def get_effective_membership_level(self, team: Team) -> Optional[int]:
+        requesting_parent_membership: OrganizationMembership = OrganizationMembership.objects.only("id", "level").get(
+            organization_id=team.organization_id, user=self.context["request"].user
         )
+        if not settings.EE_AVAILABLE:
```

Comment:
Should you check here too that project-based permissioning is enabled for this project?